templates/dashboard/mailbox.html

```diff
@@ -4,155 +4,173 @@
 {% block title %}Mailboxes{% endblock %}
 {% block default_content %}
 
-  <div class="row">
-    <div class="col">
-      <h1 class="h3">
-        Mailboxes
-        <a class="ml-3 text-info"
-           style="font-size: 12px"
-           data-toggle="collapse"
-           href="#howtouse"
-           role="button"
-           aria-expanded="false"
-           aria-controls="collapseExample">
-          How to use <i class="fe fe-chevrons-down"></i>
-        </a>
-      </h1>
-      {% if not current_user.is_premium() %}
+    <div class="row">
```

Comment:
do you know why the indentation changes here? To check the formatting, you can run `poetry run djlint --check templates`. To format a specific file, you can run ` djlint --reformat templates/dashboard/mailbox.html`